src/cript/nodes/primary_nodes/computation.py

```diff
@@ -142,6 +145,7 @@ def __init__(
     # ------------------ Properties ------------------
```

Comment:
I wonder if it is a good idea to keep this here, or if it is unneeded and should be removed. I can see both sides to it, and it is no big deal at all either way, but always looking out for improvements

Reply:
I am neutral on this.

Reply:
yeah same